src/sync/converters.ts

```diff
@@ -94,19 +98,23 @@ export function createUserEntity(user: BitbucketUser): BitbucketUserEntity {
 }
 
 export function createGroupEntity(group: BitbucketGroup): BitbucketGroupEntity {
-  const groupEntity: BitbucketGroupEntity = {
-    _type: BITBUCKET_GROUP_ENTITY_TYPE,
-    _class: BITBUCKET_GROUP_ENTITY_CLASS,
-    _key: `bitbucket-group: ${group.name}`, //there is no uuid in API v.1.0 objects
-    displayName: group.name,
-    name: group.name,
-    permission: group.permission,
-    autoAdd: group.auto_add,
-    slug: group.slug,
-    //could iterate members and owner for user uuids here and set properties of those
-    //but let's let those exist via relationships
-  };
-  return groupEntity;
+  const { members, ...source } = group; // remove members for raw data
```

Comment:
I always wonder about these kinds of things. Is it better to do something like this?

```suggestion
  group.members = group.members !== undefined ? 'REDACTED_BY_J1' : undefined;
```

Reply:
Perhaps we should have a `redactedRawData(group, ['members'])` that produces something like you're suggesting (`members` would never be `undefined` if it were defined, but have a value like `J1_REDACTED`).